.coveragerc

```diff
@@ -0,0 +1,6 @@
+[report]
+exclude_lines =
+    pragma: no cover
+
+    raise NotImplementedError
+    raise ValueError
```

Comment:
IMO, this should be removed. `ValueError` covers the corner cases.
Is it possible to include something like,
```
exclude_files =
       pragma: no cover

       *test*
```

Reply:
@czgdp1807 I have removed the line `raise ValueError` and  tested locally and it works fine and does not affect the coverage
```
exclude_files =
       pragma: no cover

       *test*
```
above code is not working its giving invalid syntax

Reply:
@czgdp1807 thanks for guidance I have removed the line `raise ValueError` and tested locally and it works fine
```
[run]
parallel = True
source = pydatastructs
omit =
    */tests/*
```
I have added the above code to exclude the tests in `.coveragerc` and merged `coveragerc_travis` in `.coveragerc`